packages/react-native-bpk-component-progress/src/BpkProgress.js

```diff
@@ -77,6 +79,12 @@ const BAR_TYPES = {
   bar: 'bar',
 };
 
+const getStyles = (
```

Comment:
Can we remove this and use the `unpack` directly inline. That way flow recognises the type correctly and converting to any is not required

Reply:
Good catch 👍